packages/serverless/src/backend_connector.js

```diff
@@ -203,6 +203,13 @@ function getBackendTimeout(localUseLambdaExtension) {
 }
 
 function send(resourcePath, payload, finalLambdaRequest, callback) {
+  let callbackWasCalled = false;
+  const handleCallback = args => {
+    if (callbackWasCalled) return;
+    callbackWasCalled = true;
+    callback(args);
+  };
```

Comment:
Alternatively, the @instana/core utility function https://github.com/instana/nodejs/blob/main/packages/core/src/util/atMostOnce.js#L28 could be used here to wrap the original callback. This has exactly the same effect. (I don't have any opinion in either direction, just wanted to mention it.)

Reply:
Good idea - will replace 👍